### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by Hosiec** [ONSdigital/design-system#2897] [Feedback]
What options do users who want to provide feedback have if they do not have a GIThub account? carla.hosie@ons.gov.uk

**Comment by daniellecorke** [comment]
> What options do users who want to provide feedback have if they do not have a GIThub account? [carla.hosie@ons.gov.uk](mailto:carla.hosie@ons.gov.uk)

The user can click the link within the banner at the top of the page that reads "This is a new service. To help us improve it, [give feedback]". This allows them to send feedback via email.
